src/OrbitCaptureClient/CaptureClient.cpp

```diff
@@ -65,10 +65,11 @@ Future<ErrorMessageOr<CaptureListener::CaptureOutcome>> CaptureClient::Capture(
     ThreadPool* thread_pool, const ProcessData& process,
     const orbit_client_data::ModuleManager& module_manager,
     absl::flat_hash_map<uint64_t, FunctionInfo> selected_functions,
-    TracepointInfoSet selected_tracepoints, absl::flat_hash_set<uint64_t> frame_track_function_ids,
```

Comment:
Should this be part of the commit? You are removing the frame_track_function_ids, right?

Reply:
yep - because they are no longer used in the client. which kind of makes sense - they after all purely user-defined and interface presentation-related.

Reply:
could you add this to the commit message?

Reply:
done.

Reply:
Was it already pushed?